### ISSUE TITLE
Crash if target account has no contact info

### ISSUE THREAD

**Issue by trysdyn** [cheeplusplus/fa-export-cli#5]
```
Got top level fatal error TypeError: Cannot read properties of undefined (reading 'next')
    at Object.convert (/home/trysdyn/nobackup/fa-export-cli/build/main.js:44001:46)
    at /home/trysdyn/nobackup/fa-export-cli/build/main.js:43443:28
    at iterateObject (/home/trysdyn/nobackup/fa-export-cli/build/main.js:43176:15)
    at handleDataObj2 (/home/trysdyn/nobackup/fa-export-cli/build/main.js:43384:9)
    at /home/trysdyn/nobackup/fa-export-cli/build/main.js:43401:26
    at iterateObject (/home/trysdyn/nobackup/fa-export-cli/build/main.js:43176:15)
    at handleDataObj2 (/home/trysdyn/nobackup/fa-export-cli/build/main.js:43384:9)
    at module2.exports [as scrapeHTML] (/home/trysdyn/nobackup/fa-export-cli/build/main.js:43450:14)
    at _FurAffinityClient.scrape (/home/trysdyn/nobackup/fa-export-cli/build/main.js:45095:45)
    at _FurAffinityClient.getUserPage (/home/trysdyn/nobackup/fa-export-cli/build/main.js:43864:27)
```

This maps to...

```
              "artist_information": {              
                "listItem": "#userpage-contact-item div.table-row",   
```

Replication confirmed against my personal FA account (dev knows what it is), and multiple other accounts with just no contact info.

Monkeypatching this in my local for now so I can grab some things of urgency.

Replicated on Node 22.5.1 and 22.6.0

**Comment by trysdyn** [comment]
This is a bug specific to the Modern site theme. Switching to Classic mitigates.

**Comment by cheeplusplus** [comment]
This should be fixed in v0.0.4, fixed upstream in https://github.com/cheeplusplus/fa.js/commit/023a101d5cf796315b7445e3a526dd0843344b59
